src/main/resources/templates/errors/500.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="https://www.thymeleaf.org"
+      xmlns:sec="https://www.thymeleaf.org/thymeleaf-extras-springsecurity3">
+
+<head>
+    <meta charset="UTF-8">
+    <title>500 - Internal Server Error</title>
+</head>
+<body>
+<h1>500 - Internal Server Error</h1>
+<h2>Check the WebSecurityConfig or Spring Security(either Thymeleaf,Beans or Annotations)</h2>
```

Comment:
Please remove this line. Because 500 may not be relating to security or WebSecurityConfig